src/AzureAuth/Commands/Ado/Pat/CommandScopes.cs

```diff
@@ -0,0 +1,46 @@
+
+using McMaster.Extensions.CommandLineUtils;
+
+using Microsoft.Authentication.AdoPat;
+using Microsoft.Extensions.Logging;
+using Microsoft.Office.Lasso.Telemetry;
+
+using System.Linq;
+
+namespace Microsoft.Authentication.AzureAuth.Commands.Ado.Pat

```

Comment:
In order to make this functionality available, we will need to release a new version of AzureAuth. This leads to the whole OMR update churn that our team has to deal with. I am assuming this is not a breaking change? I will test and verify shortly too. :)

Reply:
Correct, it is a new sub-command and not one that does authentication. This is not a breaking change, nor is it urgent. I would be happy to help with the update if needed.